recipes/gclib/meta.yaml

```diff
@@ -1,15 +1,19 @@
+{% set name = "gclib" %}
+{% set version = "0.0.1" %}
+{% set sha256hash = "8a63c4f507cfcc579c60fc9540bdce77b64b5050118f9313b12fb6e2ae5f23fe" %}
+
 package:
-  name: gclib
-  version: "0.0.1"
+  name: {{ name }}
+  version: {{ version }}
 
 build:
-    number: 4
+    number: 5
```

Comment:
I'm not sure increase the build number is correct here. We should increase the actual version.